### DISCUSSION TITLE
Selectie van contacten op een Google Maps zetten

### DISCUSSION THREAD

**Discussion by deraedta** [DaktaDeo/Multipass#2] [Ideas]
Voorbeeld zie: https://drive.google.com/open?id=1uKWBKHDcuf9kAsVl5rVq0lzs7DweGI7q&usp=sharing
Vanuit Multipass een selectie kunnen maken van contacten om op regelmatige basis Google Maps te kunnen bijwerken.
In de gratis versie van Google Maps is dat zonder API.
Het gebeurt via een importeren van een Google Sheets.

**Comment by Gompje** [comment]
lijkt me een zeer goed idee @deraedta; ik kijk alvast na of dit haalbaar is

**Comment by deraedta** [comment]
Aanvulling: iemand uit je netwerk vraagt "ken je een boekhouder uit de buurt", en dan wordt die op een kaartje getoond. Dus enkel de boekhouders uit je eigen CRM.
